server/routerlicious/packages/lambdas/src/nexus/connect.ts

```diff
@@ -79,7 +79,7 @@ function composeConnectedMessage(
 	const connectedMessage: IConnected = {
 		claims,
 		clientId,
-		existing: false,
+		existing: true,
```

Comment:
Why this change? Seems default value changed and just add a double safety confirmation

Reply:
`true` is the value in main
https://github.com/microsoft/FluidFramework/blob/main/server/routerlicious/packages/lambdas/src/nexus/index.ts#L597